### ISSUE TITLE
mods:identifier identifiers are not being generated in the METS

### ISSUE THREAD

**Issue by tomwrobel** [tomwrobel/ora_data_model#48]
https://ora4-prd-symp.bodleian.ox.ac.uk/sword/collections/default/works/db78tc00b

The MODs identifiers for this record should be

```
                    <mods:identifier type="doi">10.1234/56789</mods:identifier>
                    <mods:identifier type="eissn">1011-1212</mods:identifier>
                    <mods:identifier type="issn">1045-6007</mods:identifier>
                    <mods:identifier type="pubs_id">97575</mods:identifier>
                    <mods:identifier type="tinypid">pubs:97575</mods:identifier>
```

None of these are appearing in the record METS

In addition, see #45 on making sure that the pid is present (mods:identifier type="pid")

**Comment by anusharanganathan** [comment]
Hmm this should work. See code https://github.com/CottageLabs/willow_sword/blob/feature/ora_customizations/app/services/willow_sword/crosswalk_to_mods.rb#L168-L199. I will check this later

**Comment by anusharanganathan** [comment]
Fixed. See commit https://github.com/CottageLabs/willow_sword/commit/4636e3db92e37f5a487878f5e74e4904771bf139. Couldn't find the field in ORA model for pubs_id, so commented it out (see [line](https://github.com/CottageLabs/willow_sword/commit/4636e3db92e37f5a487878f5e74e4904771bf139#diff-4bfa481fe04fd7419a5454aca4531424R137)). Also, doi has been set as multi-valued while all other identifiers are single valued. I have added code to treat doi as multi-valued.
